hydractl/hydractl.py

```diff
@@ -443,6 +443,9 @@ def main(argv):
         else:
             jobsetdata['enabled'] = '1'
 
+        if args['edrch'] == True:
+            jobsetdata['enable_dynamic_run_command'] = '1'
```

Comment:
'1' in few lines above is for a multiple choice option. I suspect that this should be "on"
Just like in the project side: `projectdata['enable_dynamic_run_command'] = "on"`
Please check.

Reply:
At least this is the form that passed my testing.  I'm not ruling out the possibility that the hydra side parser supports BOTH '1' and 'on' (and even 'true'). Do you want me to test?

Reply:
Whatever way is used when web page is transmitted, I would use that, even if parser accepts anything resembling a true value.
And I suspect web interface uses "on" for the boolean values.

Reply:
If I get this right... sending the form from the UI seems to collect the data, and then JSON.stringify() it.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify example shows Boolean(false) to be returned as false.

Reply:
btw. I originally used '1' as the same value that the documentation tells one to use in the server level configuration.

Reply:
I have checked what the web UI is actually sending to the hydra web server when a form is filled.

If that is wrong, then maybe the other items should be changed too, for consistency.
Of course  that is not the point of this commit, but maybe worthy of an issue/ticket or something.

Reply:
Oh, you had accepted the old version while I were testing the changes.

Changed that value to 'true', and also added a comment about the implementation (reason not to change it to a form that I had at one point)

Reply:
Err... going to correct that code comment next.